content/v2.0/monitor-alert/templates/AWS.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Monitor Amazon Web Services (AWS)
+seotitle: Monitor Amazon Web Services (AWS)
+description: >
+  Use the AWS CloudWatch Monitoring template to monitor data from Amazon Web Services (AWS), Amazon Elastic Compute Cloud (EC2), and Amazon Elastic Load Balancing (ELB) with the AWS CloudWatch Service.
+menu:
+  v2_0:
+    parent: Monitor with templates
+weight: 101
+---
+
+Use the AWS CloudWatch Monitoring template to monitor data from Amazon Web Services (AWS), Amazon Elastic Compute Cloud (EC2), and Amazon Elastic Load Balancing (ELB) with the AWS CloudWatch Service.
+
+When you install this template, you'll get the following:
+
+- 2 dashboards:
+  - **AWS CloudWatch NLB (Network Load Balancers) Monitoring**: Displays data from the cloudwatch_aws_network_elb measurement
```

Comment:
- AWS CloudWatch Network Load Balancers (NLB) Monitoring
- Format "cloudwatch_aws_network_elb" as inline code.

Reply:
template is actually called AWS CloudWatch NLB Monitoring, hence the spell out part in parens